redfish-core/lib/power_subsystem.hpp

```diff
@@ -25,6 +25,47 @@ namespace redfish
 inline void getPowerSubsystemAllocation(
     const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
 {
+    // Get max from CapLimits
+    sdbusplus::asio::getAllProperties(
+        *crow::connections::systemBus, "org.open_power.OCC.Control",
+        "/xyz/openbmc_project/control/host0/power_cap_limits",
+        "xyz.openbmc_project.Control.Power.CapLimits",
+        [asyncResp](const boost::system::error_code& ec,
+                    const dbus::utility::DBusPropertiesMap& propertiesList) {
```

Comment:
Similar to above the long lambda and hard-coded path will likely not be accepted upstream.

But right now this is not yet implemented upstream. There is an abandoned commit that started trying to do this: https://gerrit.openbmc.org/c/openbmc/bmcweb/+/68442

If we revive that commit we will have to restructure this code to merge upstream. I'm okay with addressing both of these if/when that is done.